web/profiles/custom/yalesites_profile/modules/custom/ys_core/ys_core.module

```diff
@@ -313,3 +313,18 @@ function ys_core_preprocess_image_widget(&$variables) {
 function ys_core_taxonomy_term_update() {
   Cache::invalidateTags(['rendered']);
 }
+
+/**
+ * Implements hook_field_widget_complete_form_alter().
+ *
+ * This changes the label text of the link widget for cta_banners to
+ * "Call-to-action text".
+ */
+function ys_core_field_widget_complete_form_alter(&$field_widget_complete_form, FormStateInterface $form_state, $context) {
```

Comment:
Yes, let's do.  Mike did say that if we thought this would create some waves, we could leave the text as is and the overall header text for the link widget was a good enough change.  That might be our best bet so this doesn't grow over time, eliminating this hook.

Reply:
Removed it; we should be good to go then.  Thanks for talking it out with me.